common/static/common/js/spec/components/search_field_spec.js

```diff
@@ -21,8 +21,7 @@ define([
                     current_page: 1,
                     start: 0,
                     results: _.first(results, perPage)
-                },
-                {parse: true}
```

Comment:
@muzaffaryousaf [Backbone.Collection](https://cdn.rawgit.com/jashkenas/backbone/1.0.0/docs/backbone.html#section-73) takes two only parameters, Here the third parameter is extra and it has no effect.